cmake/Install.cmake

```diff
@@ -41,15 +41,6 @@ set(extra_confdir
     CACHE STRING "Path for extra configuration")
 
 
-corrosion_set_env_vars(${fish_rust_target}
-    "PREFIX=${prefix}"
-    # Temporary hack to propogate CMake flags/options to build.rs.
-    "CMAKE_WITH_GETTEXT=${CMAKE_WITH_GETTEXT}"
-    "DOCDIR=${CMAKE_INSTALL_FULL_DOCDIR}"
-    "DATADIR=${CMAKE_INSTALL_FULL_DATADIR}"
-    "SYSCONFDIR=${CMAKE_INSTALL_FULL_SYSCONFDIR}"
-    "BINDIR=${CMAKE_INSTALL_FULL_BINDIR}"
```

Comment:
@faho do we need this? Maybe for builds that use a custom `SYSCONFDIR`?

Reply:
Sorry, I don't quite recall the context here.

I *think* it was an issue with fish not finding the config when run from the build directory?

Reply:
I cannot find an issue, I can both run fish from the build directoy (and `$__fish_sysconf_dir` is effectively `../etc`), and I can `sudo ninja install` (and `$__fish_sysconf_dir` is /usr/local/etc with a prefix of /usr/local).